### PR TITLE
Add Scheduled Tasks results inbox and Home surfacing

### PR DESCRIPTION
**Pull request by rmusser01**
## Summary
- Adds the Scheduled Tasks Results experience, result projection/linking, detail drawers, and source-agnostic empty/capability states.
- Surfaces scheduled-task result and failure signals on Companion Home through a dedicated Automation Inbox that works without personalization.
- Adds notification/result dedupe, overview/task cross-links, route aliases, UX polish, responsive verification notes, and Backlog closeout records.

## Test Plan
- [x] `./node_modules/.bin/vitest run src/components/Option/ScheduledTasks/__tests__ src/components/Option/CompanionHome/__tests__ src/services/__tests__/notifications.test.ts` from `apps/packages/ui` passed: 19 files, 181 tests.
- [x] `git diff --check` passed.
- [x] Browser smoke covered `/scheduled-tasks`, `/scheduled-tasks?tab=results`, and `/companion` at desktop and 390px with temporary local mock API responses.
- [x] Bandit skipped: frontend/docs/Backlog-only changes, no Python/backend files touched.

## Change summary
Human-authored Change summary required before marking ready for review/merge per `Docs/superpowers/AI_GENERATED_PR_CHANGE_SUMMARY_POLICY_2026_04_17.md`.

---
## Summary by cubic
Adds a Scheduled Tasks Results experience and a Home Automation Inbox so users can see automation outcomes, triage failures, and deep‑link to exact runs or results. Uses projected signals from the task list and notifications; frontend‑only.

- **New Features**
  - Results tab with a results panel, filters, capability modes, and a result detail drawer; supports success, failure, running, and completed‑no‑results signals with deep links via `task_id`, `run_id`, and `result_id`.
  - Route alias `/scheduled-tasks/results` for web, extension, and hosted Next.js; route state parsing and tests included.
  - Overview, Task table, and task detail drawer surface newest signals and provide cross‑links into Results.
  - Companion Home adds an Automation Inbox card showing recent scheduled‑task results and failures from tasks and notifications; notification targets normalized/deduped and links resolved to the best result/run/task page; types extended and tests added.

- **Bug Fixes**
  - Fixed `/scheduled-tasks/results` alias navigation edge case and strengthened the hosted‑alias test to verify the module target.
  - Corrected projected‑mode copy and state labels so paused items don’t show running language.
  - Cleaned up UI props and minor link/accessibility polish in Scheduled Tasks components.

<sup>Written for commit c9a47c915e0c0159893842d9bc83782d66c3287a. Summary will update on new commits.</sup>

<a href="https://cubic.dev/pr/rmusser01/tldw_server/pull/2328?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

### CODE REVIEW

File: apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskResultsPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+    ownerFilter,
+    reviewStateFilter
+  })
+
+  const columns: ColumnsType<ScheduledTaskResultItem> = [
+    {
+      title: "Result",
+      key: "result",
+      render: (_, result) => (
+        <Space orientation="vertical" size={4}>
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskResultsPanel.tsx:208]: Verified against the installed AntD runtime in this repo: changing these `Space` usages to `direction` emits `Warning: [antd: Space] direction is deprecated. Please use orientation instead.` `orientation` is the supported prop for this version, so I retained it and am resolving this as a false positive.

File: apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskDetailDrawer.tsx [hunk 2]
```diff
@@ -163,7 +166,16 @@ export const ScheduledTaskDetailDrawer: React.FC<ScheduledTaskDetailDrawerProps>
 
           <div>
             <Typography.Title level={5}>Actions</Typography.Title>
-            {renderTaskActions({ task, onEditReminder, onDeleteReminder })}
+            <Space orientation="vertical" size={8}>
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskDetailDrawer.tsx:169]: Verified against the installed AntD runtime in this repo: changing these `Space` usages to `direction` emits `Warning: [antd: Space] direction is deprecated. Please use orientation instead.` `orientation` is the supported prop for this version, so I retained it and am resolving this as a false positive.

File: apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskOverview.tsx [hunk 3]
```diff
@@ -116,6 +122,23 @@ export const ScheduledTaskOverview: React.FC<ScheduledTaskOverviewProps> = ({
           </OverviewPanel>
         </Col>
       </Row>
+      {latestResult ? (
+        <Card size="small" style={{ marginTop: 12 }}>
+          <Space orientation="vertical" size={6} style={{ width: "100%" }}>
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskOverview.tsx:127]: Verified against the installed AntD runtime in this repo: changing these `Space` usages to `direction` emits `Warning: [antd: Space] direction is deprecated. Please use orientation instead.` `orientation` is the supported prop for this version, so I retained it and am resolving this as a false positive.

File: apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskResultDetailDrawer.tsx [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+  return (
+    <Drawer
+      aria-labelledby={titleId}
+      title={<span id={titleId}>{drawerTitle}</span>}
+      open={open}
+      onClose={onClose}
+      size={560}
+    >
+      {result ? (
+        <Space orientation="vertical" size="large" style={{ width: "100%" }}>
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskResultDetailDrawer.tsx:95]: Verified against the installed AntD runtime in this repo: changing these `Space` usages to `direction` emits `Warning: [antd: Space] direction is deprecated. Please use orientation instead.` `orientation` is the supported prop for this version, so I retained it and am resolving this as a false positive.

File: apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskResultsPanel.tsx [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+        >
+          Open signal
+        </Button>
+      )
+    }
+  ]
+
+  if (taskCount === 0) {
+    return (
+      <Space orientation="vertical" size={16} style={{ width: "100%" }}>
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskResultsPanel.tsx:253]: Verified against the installed AntD runtime in this repo: changing these `Space` usages to `direction` emits `Warning: [antd: Space] direction is deprecated. Please use orientation instead.` `orientation` is the supported prop for this version, so I retained it and am resolving this as a false positive.

File: apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskResultsPanel.tsx [hunk 6]
```diff
@@ -0,0 +1,402 @@
...
+              </Typography.Text>
+            </Space>
+          }
+        />
+      </Space>
+    )
+  }
+
+  return (
+    <Space orientation="vertical" size={16} style={{ width: "100%" }}>
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskResultsPanel.tsx:294]: Verified against the installed AntD runtime in this repo: changing these `Space` usages to `direction` emits `Warning: [antd: Space] direction is deprecated. Please use orientation instead.` `orientation` is the supported prop for this version, so I retained it and am resolving this as a false positive.

File: apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskResultsPanel.tsx [hunk 7]
```diff
@@ -0,0 +1,402 @@
...
+        />
+      )}
+    </Space>
+  )
+}
+
+const PanelHeader: React.FC<{ capabilityMode: ScheduledTaskResultsCapabilityMode }> = ({
+  capabilityMode
+}) => (
+  <Space orientation="vertical" size={12} style={{ width: "100%" }}>
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTaskResultsPanel.tsx:379]: Verified against the installed AntD runtime in this repo: changing these `Space` usages to `direction` emits `Warning: [antd: Space] direction is deprecated. Please use orientation instead.` `orientation` is the supported prop for this version, so I retained it and am resolving this as a false positive.

File: apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTasksPage.tsx [hunk 8]
```diff
@@ -361,13 +404,22 @@ export const ScheduledTasksPage: React.FC = () => {
...
+  const hasResultRouteTarget =
+    routeState.tab === "results" &&
+    Boolean(routeState.resultId || routeState.runId || routeState.taskId)
+  const missingRouteResult =
+    hasLoadedTasks &&
+    hasResultRouteTarget &&
+    selectedResult === null
 
   const renderOverviewTab = () => (
     <Space orientation="vertical" size={16} style={{ width: "100%" }}>
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTasksPage.tsx:423]: Verified against the installed AntD runtime in this repo: changing these `Space` usages to `direction` emits `Warning: [antd: Space] direction is deprecated. Please use orientation instead.` `orientation` is the supported prop for this version, so I retained it and am resolving this as a false positive.

File: apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTasksPage.tsx [hunk 9]
```diff
@@ -381,6 +433,34 @@ export const ScheduledTasksPage: React.FC = () => {
...
+      taskId: result.taskId
+    })
+  }
+
+  const openTaskResults = (task: ScheduledTask) => {
+    updateRoute({ tab: "results", taskId: task.id })
+  }
+
+  const renderResultsTab = () => (
+    <Space orientation="vertical" size={16} style={{ width: "100%" }}>
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTasksPage.tsx:457]: Verified against the installed AntD runtime in this repo: changing these `Space` usages to `direction` emits `Warning: [antd: Space] direction is deprecated. Please use orientation instead.` `orientation` is the supported prop for this version, so I retained it and am resolving this as a false positive.

File: apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTasksPage.tsx [hunk 10]
```diff
@@ -381,6 +433,34 @@ export const ScheduledTasksPage: React.FC = () => {
...
+        taskCount={tasks.length}
+        capabilityMode="projected_signals"
+        onCreateTask={openCreateReminder}
+        onOpenResult={openResultSignal}
+      />
+    </Space>
+  )
+
   const renderTasksTab = () => (
     <Space orientation="vertical" size={16} style={{ width: "100%" }}>
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTasksPage.tsx:472]: Verified against the installed AntD runtime in this repo: changing these `Space` usages to `direction` emits `Warning: [antd: Space] direction is deprecated. Please use orientation instead.` `orientation` is the supported prop for this version, so I retained it and am resolving this as a false positive.

File: apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTasksPage.tsx [hunk 11]
```diff
@@ -58,21 +66,34 @@ export const ScheduledTasksPage: React.FC = () => {
...
   const routeState = React.useMemo(
-    () => parseScheduledTaskRouteState(searchParams),
-    [searchParams]
+    () =>
+      parseScheduledTaskRouteState(searchParams, {
+        defaultTab: location.pathname.replace(/\/+$/, "").endsWith("/scheduled-tasks/results")
+          ? "results"
+          : "overview"
+      }),
+    [location.pathname, searchParams]
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTasksPage.tsx:75]: Fixed in `3f9fed2cca` by normalizing `/scheduled-tasks/results` back to `/scheduled-tasks` when Overview is selected, with regression coverage for navigating from the alias path back to Overview.

[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/ScheduledTasksPage.tsx:75]: Fixed in `3f9fed2cca` by normalizing `/scheduled-tasks/results` back to `/scheduled-tasks` when Overview is selected, with regression coverage for navigating from the alias path back to Overview.

File: apps/packages/ui/src/components/Option/ScheduledTasks/scheduled-task-results.ts [hunk 12]
```diff
@@ -0,0 +1,809 @@
...
+    const countText =
+      resultCount !== null
+        ? `${resultCount} ${resultCount === 1 ? "result" : "results"}`
+        : "results"
+    return sourceLabel ? `Found ${countText} from ${sourceLabel}.` : `Found ${countText}.`
+  }
+
+  if (signalKind === "running") {
+    return `${task.title} is running now.`
+  }
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/ScheduledTasks/scheduled-task-results.ts:367]: Fixed in `3f9fed2cca` by checking `state === "paused"` before the running branch in the summary/status helpers. Added projector coverage for paused copy and Results panel coverage for the visible paused signal.

File: apps/packages/ui/src/routes/__tests__/scheduled-tasks-route.test.tsx [hunk 13]
```diff
@@ -52,6 +54,12 @@ describe("scheduled tasks route wiring", () => {
     expect(extensionRouteRegistrySource).toContain('path: "/scheduled-tasks"')
   })
 
+  it("registers scheduled tasks Results aliases across route surfaces", () => {
+    expect(webRouteRegistrySource).toContain('path: "/scheduled-tasks/results"')
+    expect(extensionRouteRegistrySource).toContain('path: "/scheduled-tasks/results"')
+    expect(existsSync(hostedResultsAliasPath)).toBe(true)
+  })
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/routes/__tests__/scheduled-tasks-route.test.tsx:63]: Fixed in `3f9fed2cca` by reading the hosted alias module and asserting it targets `option-scheduled-tasks`, so the test no longer passes on file existence alone.

File: backlog/tasks/task-2338 - Implement-Scheduled-Tasks-Phase-3-Home-Automation-Inbox-surfacing.md [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+- Bandit: skipped because this slice only changes frontend TypeScript/React and Backlog/plan text, not Python executable code.
+<!-- SECTION:NOTES:END -->
+
+## Final Summary
+
+<!-- SECTION:FINAL_SUMMARY:BEGIN -->
+Implemented Scheduled Tasks Phase 3 Home surfacing. Home now includes a fixed Automation Inbox module that can show scheduled-task result and failure signals without Companion personalization, merge projected task signals with notification-derived signals, handle partial failures without blocking existing Home cards, and deep-link to exact Scheduled Tasks result/run/task targets.
+<!-- SECTION:FINAL_SUMMARY:END -->
+
+<!-- SECTION:FINAL_SUMMARY:END -->
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply backlog/tasks/task-2338 - Implement-Scheduled-Tasks-Phase-3-Home-Automation-Inbox-surfacing.md]: Fixed in `3f9fed2cca` by removing the duplicate `<!-- SECTION:FINAL_SUMMARY:END -->` marker from TASK-2338. `rg` now shows one final-summary end marker for that task.

File: apps/packages/ui/src/components/Option/CompanionHome/hooks.ts [hunk 15]
```diff
@@ -156,6 +165,104 @@ export const useCompanionHomeData = ({
...
+
+      let nextError: string | null = null
+      if (tasksResult.status === "rejected") {
+        nextError =
+          notificationItems.length > 0
+            ? "Some scheduled-task signals could not be loaded."
+            : "Automation signals unavailable"
+      } else if (notificationsResult.status === "rejected") {
+        nextError = "Recent automation notifications could not be loaded."
+      } else if (tasksResult.value.partial) {
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/CompanionHome/hooks.ts]: Fixed in `3f9fed2cca` by guarding `tasksResult.value` with optional chaining and defaulting projected task items to an empty list when the fulfilled payload is unexpectedly absent.

File: apps/packages/ui/src/components/Option/CompanionHome/hooks.ts [hunk 16]
```diff
@@ -156,6 +165,104 @@ export const useCompanionHomeData = ({
...
+      const notificationItems =
+        notificationsResult.status === "fulfilled"
+          ? buildScheduledTaskAutomationHomeItemsFromNotifications(
+              notificationsResult.value.items
+            )
+          : []
+      const nextPartial =
+        tasksResult.status === "rejected" ||
+        notificationsResult.status === "rejected" ||
+        (tasksResult.status === "fulfilled" && tasksResult.value.partial)
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/CompanionHome/hooks.ts]: Fixed in `3f9fed2cca` by guarding `tasksResult.value` with optional chaining and defaulting projected task items to an empty list when the fulfilled payload is unexpectedly absent.

File: apps/packages/ui/src/components/Option/CompanionHome/cards/AutomationInboxCard.tsx [hunk 17]
```diff
@@ -0,0 +1,127 @@
...
+    ? "Checking recent scheduled-task results and notifications."
+    : error && !hasItems
+      ? "Scheduled-task results are temporarily unavailable. Other Home cards remain available."
+      : "Results and failures from scheduled tasks appear here after a run."
+
+  return (
+    <section className="rounded-3xl border border-border/80 bg-surface/90 p-5 shadow-sm backdrop-blur-sm">
+      <div className="flex items-start justify-between gap-3">
+        <div>
+          <h2 className="text-lg font-semibold text-text">Automation Inbox</h2>
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/CompanionHome/cards/AutomationInboxCard.tsx]: Fixed in `3f9fed2cca` by wrapping the Automation Inbox user-visible strings in `useTranslation` calls with default copy, using the same object-overload pattern as the rest of the UI package.
